presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -248,6 +248,9 @@ public void testReplace()
         assertFunction("CAST(REPLACE('abc' || utf8(from_hex('CE')) || 'xyz', '', 'X') AS VARBINARY)",
                 VARBINARY,
                 new SqlVarbinary(new byte[] {'X', 'a', 'X', 'b', 'X', 'c', 'X', (byte) 0xCE, 'X', 'x', 'X', 'y', 'X', 'z', 'X'}));
+
+        String longString = new String(new char[1_000_000]).replace("\0", "a");
```

Comment:
Why "\0"? Shouldn't it be just ""?

Reply:
`new String(new char[1_000_000])` makes a string of 1M characters, each is `\0`.

Reply:
Oh - I got mixed up between Java replace and sql replace :) Actually you should not need this replace at all because arrays are initialized to default value. So just leave it as is or may be use nCopies or something

Reply:
@kaikalur You are right. I don't need this replace. I'll change it to `String longString = new String(new char[1_000_000]);`